### DISCUSSION TITLE
BottomSheetScroolView doesn't work on IOS

### DISCUSSION THREAD

**Discussion by RomanSuvorov** [gorhom/react-native-bottom-sheet#738] [Q&A]
When i try scroll BottomSheetScrollView or BottomSheetSectionList nothing happens. I can't pull up bottomSheet (only if tap and drag by outer space of scrollList inside BottomSheet)

![image](https://user-images.githubusercontent.com/20489656/141084268-4b5ffb63-8db0-4f29-9b03-e428df8c2c15.png)

If tap on 1 --- all work fine, but when tap on 2 element --- nothing happen (BottomSheet doesn't pull up and scrollList doesn't scroll)

On Android all works fine. When bottomSheet at last snapPoint, I can scroll interior list

**Comment by RomanSuvorov** [comment]
Guys, please tell me, is this a bug or is this functionality not yet available on IOS?
I can share video demonstration or code snippet, if it's needed

**Comment by TheBestMoshe** [comment]
Possibly related to https://github.com/gorhom/react-native-bottom-sheet/issues/800?
